generatedSchema.ts

```diff
@@ -0,0 +1,31 @@
+/** WORK IN PROGRESS, THIS FILE WILL BE AUTOGENERATED */
```

Comment:
Since it will be a generated file, it should not be committed at all. I'll need to update the `package.json` scripts to facilitate this and at some point I'm going to be working (with low priority) on a type generator for the `ERC725JSONSchema`. 

Typescript users will be able to generate types that they will be able to pass in as follows:
```typescript
new ERC725<SchemaTypes>
``` 

This will result in pretty nice IDE code-completion for all the main functions. As far as I can tell, that is also going to solve #13 when it's completed.